pkgs/development/python-modules/opentracing/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "opentracing";
+  version = "2.2.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "cfd231ba5c58f90bc277787e62861eb0c6e4af76e42957bec240bbdf71fb7e0e";
+  };
+
+#  preInstall = ''
+#    virtual-env env
+#    ../env/bin/activate
+#    make bootstrap
+#    make test
+#  '';
+
+  doCheck = false;
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/opentracing/opentracing-python;
+    description = "This library is a Python platform API for OpenTracing";
```

I would also shorten this, we already know it's a python library
```suggestion
    description = "Platform API for OpenTracing";
```

---

> I would also shorten this, we already know it's a python library

OK, I will update this with my next commit.

---

> I would also shorten this, we already know it's a python library
Done.